frontend/lib/identification/card_detail_view/extend_card_notification.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+
+import 'package:ehrenamtskarte/build_config/build_config.dart' show buildConfig;
+import 'package:ehrenamtskarte/l10n/translations.g.dart';
+import 'package:url_launcher/url_launcher_string.dart';
+
+class ExtendCardNotification extends StatefulWidget {
+  @override
+  State<ExtendCardNotification> createState() => _ExtendCardNotificationState();
+}
+
+class _ExtendCardNotificationState extends State<ExtendCardNotification> {
+  bool _isVisible = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context).colorScheme;
+    final t = context.t;
+
+    return _isVisible
+        ? Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: Container(
+              padding: const EdgeInsets.all(16),
+              decoration: BoxDecoration(
+                color: theme.primary.withOpacity(0.1),
+                borderRadius: BorderRadius.circular(8),
+              ),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Icon(Icons.info, color: theme.primary),
+                  SizedBox(width: 16),
+                  Expanded(
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(t.identification.extendCardNotificationTitle),
+                        SizedBox(height: 8),
+                        Text(
+                          t.identification.extendCardNotificationDescription,
+                          textScaler: TextScaler.linear(0.9),
+                        ),
+                        SizedBox(height: 8),
+                        ElevatedButton(
+                          style: ElevatedButton.styleFrom(
+                            backgroundColor: theme.primary,
+                            foregroundColor: theme.inversePrimary,
+                            shape: RoundedRectangleBorder(
+                              borderRadius: BorderRadius.circular(4),
+                            ),
+                          ),
+                          onPressed: () => _openApplication(),
+                          child: Text(t.identification.extendCard.toUpperCase()),
+                        ),
+                      ],
+                    ),
+                  ),
+                  SizedBox(width: 16),
+                  GestureDetector(
```

Comment:
There is also a standard widget from flutter that you may use (optional)
```js
                  IconButton(
                    padding: EdgeInsets.all(0),
                    alignment: Alignment.topRight,
                    onPressed: () {
                      setState(() {
                        _isVisible = false;
                      });
                    },
                    icon: const Icon(Icons.close),
                  ),
```

Reply:
To make the IconButton look the same as it does now, it needs to be customized as follows:
```
                  IconButton(
                    onPressed: () {
                      setState(() {
                        _isVisible = false;
                      });
                    },
                    icon: const Icon(Icons.close),
                    iconSize: 16,
                    padding: EdgeInsets.zero,
                    constraints: const BoxConstraints(),
                    style: const ButtonStyle(
                      tapTargetSize: MaterialTapTargetSize.shrinkWrap,
                    ),
                  ),
```
While for the GestureDetector I only need to set 2 attributes.

Reply:
then keep it as it is